Core/Core/Submissions/Submission.swift

```diff
@@ -132,7 +132,7 @@ extension Submission: WriteableModel {
         model.workflowState = item.workflow_state
         model.latePolicyStatus = item.late_policy_status
         model.pointsDeducted = item.points_deducted
-        model.attempt = item.attempt ?? 1
+        model.attempt = item.attempt ?? 0
```

Comment:
@vanwagonet is there a reason we defaulted to 1 and not 0?

Reply:
lol, no.